src/main/resources/application-security.yml

```
@@ -0,0 +1,24 @@
+#            client-name: Kakao
+#
+#        provider:
+#          kakao:
+#            authorization-uri: https://kauth.kakao.com/oauth/authorize
+#            token-uri: https://kauth.kakao.com/oauth/token
+#            user-info-uri: https://kapi.kakao.com/v2/user/me
+#            user-name-attribute: id
+
+app:
+  jwt:
+    secret: RTQZVjcvwwveXp5OwjdzxGGQ669FnDhl1PJEbzcGDYo=
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)